core/src/main/java/de/bwaldvogel/mongo/backend/AbstractMongoBackend.java

```diff
@@ -123,15 +122,15 @@ public Document getServerStatus() {
         return serverStatus;
     }
 
-    private Integer getProcessId() {
+    public Integer getProcessId() {
```

Comment:
why do we need this method to be public?

Reply:
reverted